components/integrations/edit_oauth_app/edit_oauth_app.test.tsx

```diff
@@ -76,7 +95,7 @@ describe('components/integrations/EditOAuthApp', () => {
         );
 
         wrapper.setState({showConfirmModal: false});
-        wrapper.instance().handleConfirmModal();
+        (wrapper.instance() as EditOAuthApp).handleConfirmModal();
```

Comment:
Instead of using `as` here, you can declare the type of the `wrapper` when calling shallow. The difference here is pretty minor, but it does help if you're ever calling `wrapper.instance()` multiple times.
```javascript
const wrapper = shallow<EditOAuthApp>(
    <EditOAuthApp {...props}/>,
);
```